content/en/serverless/guide/extension_motivation.md

```diff
@@ -0,0 +1,54 @@
+
+| Runtime | Extension support | Forwarder support |
+| ------- | ----------------- | ----------------- |
+| Python  | Yes               | Yes               |
+| Node.js | Yes               | Yes               |
+| Go      |                   | Yes               |
+| Java    |                   | Yes               |
+| .NET    |                   |                   |
+| Ruby    |                   |                   |
+| PHP     |                   |                   |
+
+For Python and Node.js runtimes, the Lambda extension has full feature parity with the Forwarder. The Lambda extension can submit Lambda logs, Lambda traces (generated by `dd-trace`, not X-Ray), real-time enhanced Lambda metrics, and custom metrics generated by Lambda functions.
+
+### Advantages
+
+The Datadog Lambda Extension offers the following advantages over the Datadog Forwarder:
+
+- **Simple to set up**: Triggers need to be set up on the Forwarder for every new Lambda function. The Datadog Lambda extension can be easily added as a Lambda layer. Unlike with the Forwarder, configuring the Lambda extension does not require permissions to install third-party AWS CloudFormation stacks. Additionally, since the Lambda extension sends telemetry directly to Datadog, you do not need to manage CloudWatch Log group subscriptions for your Lambda functions.
+- **Less infrastructure to manage**: The Lambda extension's simpler configuration enables you to save time on infrastructure management. **Note**: You still require the Forwarder for non-Lambda AWS integrations.
+- **Skip CloudWatch Logs**: The Forwarder converts logs to metrics and traces, which are then sent to Datadog. The Datadog Lambda Extension sends traces, metrics, and logs directly to Datadog, enabling you to reduce the cost associated with CloudWatch Logs.
+
+**Note**: If you are reporting to Datadog sites in the same regions as the ones in which you are deployed (for example: if you are deployed in APAC regions and are reporting to APAC Datadog sites), your use-case is well suited for the extension. However, if you are reporting to a Datadog site in a different region than the one in which you are deployed (for example: if you are deployed in APAC regions and are reporting to US Datadog sites) the Forwarder may be a cheaper option.
+
+### Trade-offs
+
+Migration from the Forwarder to the Lambda extension can be an involved process. There is one Forwarder setup per region and account, but there is one Lambda extension setup **per function**. That is, if you are shipping data from *N* functions using the Forwarder, you must set up the Lambda extensio *N* times before the migration is complete. Since you must reconfigure and redeploy **each** function, the full process of migration may take some time.
```

Comment:
We should note that our plugins for the Serverless Framework, AWS CDK, AWS SAM, and the Datadog CLI can automate this by letting you add the extension to an application composed of many Lambda functions all at once.

Reply:
Actually don't think this trade-off holds true. Because the functions emitting data via forwarder already have the Datadog Lambda library layer installed, which also requires set up per function. Installing one more layer doesn't seem to be too much extra work.

Reply:
This holds true only if the customer has only been collecting Lambda logs using the Forwarder, aka, no lambda library installed to each lambda function -- I would recommend them continue doing so.

If the customer has installed Lambda library to their individual Lambda functions, and been collecting metrics and traces in addition to logs, I would recommend them migrate to using Extension.